pkg/build/buildkit/buildx.go

```diff
@@ -74,12 +102,15 @@ func (b *Builder) BuildInvocationImage(manifest *manifest.Manifest) error {
 	}
 
 	out := ioutil.Discard
-	if b.IsVerbose() {
-		out = b.Out
+	if b.IsVerbose() || b.Config.IsFeatureEnabled(experimental.FlagStructuredLogs) {
+		ctx, log = log.StartSpan("buildkit", attribute.String("source", "porter.build.buildkit"))
```

Comment:
I would like to have a source set for all log lines to help filter by source. It will need more experimentation though so for now I'm using it just here and we can complete that in a follow-up if it turns out to be helpful.